go/arrow/array/null.go

```diff
@@ -118,6 +118,13 @@ func (b *NullBuilder) AppendNull() {
 	b.builder.nulls++
 }
 
+func (b *NullBuilder) AppendNulls(n int) {
+	for i := 0; i < n; i++ {
+		b.builder.length++
+		b.builder.nulls++
+	}
+}
```

Comment:
would we want `AppendNulls` and `AppendEmptyValues` to be added to all builders and added to the `Builder` interface?